Lib/gzip.py

```diff
@@ -17,7 +17,12 @@
 
 READ, WRITE = 1, 2
 
-def open(filename, mode="rb", compresslevel=9,
+COMPRESS_LEVEL_FAST = 1
+COMPRESS_LEVEL_TRADEOFF = 6
+COMPRESS_LEVEL_BEST = 9
```

Comment:
Please make these constants private. If you really want to make them public, you should document them.

Reply:
maybe with a new PR/bpo